crates/ruff_linter/src/rules/pylint/rules/import_outside_top_level.rs

```diff
@@ -44,6 +44,16 @@ use crate::{
 ///     print(python.python_version())
 /// ```
 ///
+/// ## Options
+/// - `lint.flake8-tidy-imports.banned-module-level-imports`
+///
+/// ## See also
+/// This rule will ignore import statements configured in
+/// [`lint.flake8-tidy-imports.banned-module-level-imports`][banned-module-level-imports]
+/// for the rule [`banned-module-level-imports`][TID253].
```

Comment:
Wasn't sure whether to put this directly above `## Example`, in `## See also` or merge it into `## Options`.

```suggestion
/// ## Options
/// - This rule will ignore import statements configured in
/// [`lint.flake8-tidy-imports.banned-module-level-imports`][banned-module-level-imports]
/// for the rule [`banned-module-level-imports`][TID253].
```

Reply:
Also, I didn't validate, but is this true all the time or only if  `TID253` is enabled ?

Reply:
I like having a separate section, but I'd probably put `## See also` before `## Options`.

Oh, I didn't notice this when looking at the rule initially, but this _is_ only true if TID253 is enabled. Good catch! Maybe that means we can just put the whole explanation in `See also` and not have separate `Options` since they only apply conditionally. I like your current `See also` section in that case, possibly with a slight alteration to the last line:

```rust
/// if the rule [`banned-module-level-imports`][TID253] is enabled.
```